README.md

```diff
@@ -23,14 +23,75 @@ practices.
 ## Versioning
 The commit hash of the deployed code is considered its version identifier. The commit hash can be retrieved locally via `git rev-parse HEAD`.
 
-## Setting Up
+## Setting Up Your Local Dev Environment
 
 Contile uses Rust, and requires the latest stable iteration. See
 [rustup.rs](https://rustup.rs/) for how to install this application.
 
 Once Rust is installed you can compile using `cargo build`. This will
 create a development release.
 
+Additionally, Contile requires a connection to the ADM servers. These use several 
+keys (which are specified in the configuration file) to pull results.  The contract
+tests include a stub application that can provide ADM-like returns. See the 
+[adm_settings_test.json](./adm_settings_test.json) file for reference.
+
+Normally, these configurations run as a part of a `docker-compose` command, but the individual
+Docker files can be run by themselves.
+
+To start the ADM stub, you can run the following from the root of the Contile source repository:
+
+```shell
+    docker run \
+    --env PORT=5000 \
+    --env RESPONSES_DIR=/tmp/partner/ \
+    --env ACCEPTED_MOBILE_FORM_FACTORS=phone,tablet \
+    --env ACCEPTED_DESKTOP_FORM_FACTORS=desktop \
+    -v `pwd`/test-engineering/contract-tests/volumes/partner:/tmp/partner \
+    -p 5000:5000 \
+    mozilla/contile-integration-tests-partner
+```
+
+To use the above, you'll need to set the Contile configuration option as follows:
+`adm_endpoint_url`  = `http://localhost:5000/tilesp/desktop`.  Note, you also need to 
+define a SOV (Share of Voice) source file, so make sure this is provided for partner allocations. Currently, a sample file is in the contract-tests directory, so you can make use of that, or define your own allocation JSON file to point to. 
+
+It may be helpful to create a settings file to contain all of the preferred settings, like the example of `sa-test.toml` below:
+
+```toml
```

Comment:
Since this "code" is going to be put into a file anyways, what do you think about just adding a `sa-test.toml` file to the repo for easier set up?